stl/inc/type_traits

```diff
@@ -366,6 +366,18 @@ struct is_enum : bool_constant<__is_enum(_Ty)> {}; // determine whether _Ty is a
 template <class _Ty>
 _INLINE_VAR constexpr bool is_enum_v = __is_enum(_Ty);
 
+#if _HAS_CXX23
+// STRUCT TEMPLATE is_scoped_enum
+template <class _Ty, bool = conjunction_v<is_enum<_Ty>, negation<is_convertible<_Ty, int>>>>

```

Comment:
As far as I can tell this is not valid. the second template parameter is user visible